insertion_sort/Python/insertion_sortInteractive.py

```diff
@@ -0,0 +1,14 @@
+"""
+this algorithm was made based on the existing pseudocode 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpjeppxcpi/insertion_sort/Python/insertion_sortInteractive.py
+++ b/tmp/tmpjeppxcpi/insertion_sort/Python/insertion_sortInteractive.py
@@ -1,5 +1,5 @@
 """
-this algorithm was made based on the existing pseudocode 
+this algorithm was made based on the existing pseudocode
 in the 3rd version of the Cormen's book.
 """
 
```